archive.md

```diff
@@ -3,6 +3,12 @@ layout: standalone
 title: Archive
 ---
 
+<p>
+    Search the archives or browse in reverse chronological order below.
+
+    {% include search_box.html %}
+</p>
```

Comment:
i don't think we need the `<p>` tags here, do we?

if we need to adjust margin/padding, we can use something like `<div class="mt-2 mb-2">` instead 

(for margin-top and margin-bottom, can specify 1-4)